src/routes/_utils/scrollIntoView.js

```diff
@@ -0,0 +1,64 @@
+import {
+  getScrollContainer,
+  getOffsetHeight } from './scrollContainer'
+import { smoothScroll } from './smoothScroll'
+
+function getTopOverlay () {
+  return document.getElementById('main-nav').clientHeight
```

Comment:
This element can be cached

Reply:
Yes. I'll send out a fix.

Reply:
I tried to address it in #910. Let's continue the conversation there.